src/sections/Company/News-single/index.js

```diff
@@ -65,11 +66,19 @@ const NewsSingle = ({data}) => {
               </Col>
             </Row>
           </div>
-          <RelatedPosts 
+          <div style={{ display: "flex" }}>
```

Comment:
@Chadha93 okay so I think I understood what the problem is. Some news files don't have the `eurl`, mostly if they are our own articles - By`The Newsroom`. Then it makes no sense to add this. Can you add a conditional and display only if the `eurl` is present?

Reply:
Gotcha, now I see, it works with other news having an external link. I'll update it. Thanks. :+1: